.github/workflows/docker.yaml

```diff
@@ -129,3 +131,12 @@ jobs:
           docker manifest annotate ${IMAGE}:${TAG} ${IMAGE}:arm64-${TAG} --arch arm64
           docker manifest annotate ${IMAGE}:${TAG} ${IMAGE}:amd64-${TAG} --arch amd64
           docker manifest push     ${IMAGE}:${TAG}
+      - name: Create and push Docker latest manifest
+        if: ${{is_default_branch}}
```

Comment:
I think `is_default_branch` is actually specific to the docker metadata action. More generally we can accomplish this with
 ```suggestion
         if: github.ref == 'refs/heads/master'
```